bundles/framework/coordinatetool/plugin/CoordinateTransformationExtension.js

```diff
@@ -82,11 +82,11 @@ Oskari.clazz.define('Oskari.mapframework.bundle.coordinatetool.plugin.Coordinate
          _populateCoordinatesTransformSelect: function(select) {
             var me = this,
                 projections = me._config.supportedProjections;
```

Comment:
Use "me._config.supportedProjections || []" to make sure the forEach() call doesn't fail.